test/ejabberd_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -561,6 +737,22 @@ send_message(Config) ->
                 escalus:assert(is_headline_message, [<<"Subj">>, <<"Hi Bob!!">>], Stanza3)
         end).
 
+send_message_wrong_jid(Config) ->
+    escalus:story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+        {_, Err1} = ejabberdctl("send_message_chat", ["'@@#$%!!.§§£'",
+                                                   escalus_client:full_jid(Bob),
+                                                   "\"Hello bobby!\""], Config),
+        {_, Err2} = ejabberdctl("send_message_headline", ["'%%@&@&@==//\///'",
+                                                       escalus_client:short_jid(Bob),
+                                                       "Subj", "\"Are
+                                                       you there?\""],
+                             Config),
+        Err1 =/=0,
```

Comment:
I noticed this on Travis when testing stuff rebased on top of this PR:

```
ejabberdctl_SUITE.erl:750: Warning: use of operator '=/=' has no effect
ejabberdctl_SUITE.erl:751: Warning: use of operator '=/=' has no effect
```

Which makes sense, since these lines are not assertions! They evaluate to a `boolean()` which is then discarded.

Please rewrite to:

```erlang
true = Err1 =/= 0
```

or 


```erlang
?assertNotEqual(0, Err1)
```

or

```erlang
?assert(Err1 =/= 0)
```